.github/workflows/create-github-release.yml

```diff
@@ -0,0 +1,32 @@
+name: create-github-release
```

Comment:
Hi @iowillhoit - is there any documentation that helps me understand the reason of these changes and how to use the new workflow?

Reply:
Hey @EstebanRomero84, I've got a PR for docs open here https://github.com/salesforcecli/github-workflows/pull/63/files

Everything will work as it used to, the prerelease support is much more flexible with this change